src/Controls/src/Core/Internals/PropertyPropagationExtensions.cs

```diff
@@ -8,6 +8,14 @@ namespace Microsoft.Maui.Controls.Internals
 	/// <include file="../../../docs/Microsoft.Maui.Controls.Internals/PropertyPropagationExtensions.xml" path="Type[@FullName='Microsoft.Maui.Controls.Internals.PropertyPropagationExtensions']/Docs/*" />
 	public static class PropertyPropagationExtensions
 	{
+		internal static void PropagatePropertyChanged(string propertyName, Element element, IEnumerable children)
+		{
+			if (children == null)
+				return;
+
+			PropagatePropertyChanged(propertyName, element, children.OfType<IVisualTreeElement>().ToList());
+		}
+
 		internal static void PropagatePropertyChanged(string propertyName, Element element, IReadOnlyList<IVisualTreeElement> children)
```

Comment:
Can we remove internal from this one?

Reply:
removing the internal would for this to be merged on .net 10, no?

Reply:
yeah, this will have to go into net10.

But, is there a reason for this to be IReadOnlyList instead of `IEnumerable <IVisualTreeElement>`? The result is the same as we are just calling `ToArray`

Reply:
I see all the other overloads take `Element`, so why does this need to be `IVisualTreeElement`?